src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -202,57 +203,81 @@ await client.TryInvokeAsync<IRemoteAssetSynchronizationService>(
             cancellationToken).ConfigureAwait(false);
     }
 
-    private async ValueTask SynchronizeTextChangesAsync(
-        ImmutableSegmentedList<(Document oldDocument, Document newDocument)> values,
-        CancellationToken cancellationToken)
+    private async Task DispatchSynchronizeTextChangesAsync(
+        Document oldDocument,
+        Document newDocument)
     {
-        var client = await RemoteHostClient.TryGetClientAsync(_workspace, cancellationToken).ConfigureAwait(false);
-        if (client == null)
+        // Attempt to inform the remote asset synchronization service as quickly as possible
+        // about the text changes between oldDocument and newDocument. By doing this, we can
+        // reduce the likelihood of the remote side encountering an unknown checksum and
+        // requiring a synchronization of the full document contents.
+        // This method uses JTF.RunAsync to create a fire-and-forget task. JTF.RunAsync will
```

Comment:
Comment now stale? RunAsync not being used anymore?

Reply:
Removed the stale bit.